protocol/common/message.go

```diff
@@ -214,6 +214,8 @@ type Message struct {
 	ContactVerificationState ContactVerificationState `json:"contactVerificationState,omitempty"`
 
 	DiscordMessage *protobuf.DiscordMessage `json:"discordMessage,omitempty"`
+
+	JustEdited bool
```

Comment:
I assume `Edited` as property would clash with an already existing one? Or is there any other reason for this particular naming?

Reply:
The idea is that there is no point in storing this field in the database thus if I name it `Edited` it will always be `false` on fetching message from the database, although in reality it might have been actually edited already and be in conflict with `EditedAt`. That potentially might be confusing. But it seems that `JustEdited` name for a short living flag might be confusing as well... :) Let me know if you have some better options

Reply:
I see! I guess it makes sense to at least add a comment to that property then with the clarifications you've just made.

Reply:
Added the comment, thanks!